### PR TITLE
First 9 challenges translated

### PR DESCRIPTION
**Pull request by mizujin**
#### Pre-Submission Checklist

- [X] Your pull request targets the `staging` branch of FreeCodeCamp.
- [X] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [X] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [X] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [X] Add new translation (feature adding new translations)
#### Checklist:

- [ ] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX
#### Description

Related to issue no 7288, but doesn't closes it yet.

Translated to french the first nine challenges of section 01 - Front end development certification : Html5 and CSS.

### CODE REVIEW

**Comment by Bouncey** [comment]
@mizujin We need someone from [TranslationFrench](https://gitter.im/FreeCodeCamp/TranslationFrench) to read over these

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -44,6 +44,20 @@
         "Каждое испытание имеет функцию проверки решения, которую вы можете запустить в любой момент нажатием на кнопку \"Run tests\". Как только ваше решение будет удовлетворять всем требованиям, вы сможете перейти к следующему испытанию.",
         "Для прохождения этого испытания измените текст элемента <code>h1</code> с \"Hello\" на \"Hello World\". Затем нажмите кнопку \"Run tests\"."
       ],
+      "titleFR": "Dites bonjour aux éléments HTML",
+      "descriptionFR": [
+        "Bienvenue au premier défi de code de Free Code Camp.",
+        "Vous pouvez modifier le <code>code</code> dans votre <code>éditeur texte</code>, lequel est disponible dans cette page.",
+        "Voyez-vous le code dans l'éditeur texte qui dit <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? C'est un <code>élément</code> HTML.",
+        "La plupart des éléments HTML on une <code>balise ouvrante</code> et une <code>balise fermante</code>",
```
Comment (Tagada85) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Small typo : 'on' should read 'ont' .

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 2]
```diff
@@ -160,6 +182,13 @@
         "<code>&#60;p&#62;Я метка p!&#60;/p&#62;</code>",
         "Создайте элемент <code>p</code> под вашим элементом <code>h2</code> и присвойте ему текст \"Hello Paragraph\"."
       ],
+      "titleFR": "Informez avec l'élément paragraphe",
+      "descriptionFR": [
+        "l'élément <code>p</code> est le préféré pour les textes de paragraphes de taille normale. P est l'accronyme de \"paragraphe\",",
```
Comment (Tagada85) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Just missing an uppercase L at the start of the description.

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 3]
```diff
@@ -314,7 +356,14 @@
...
         "5 веков довольно долгий период. Так как мы строим приложение CatPhotoApp, давайте используем нечто называемое текстом <code>kitty ipsum</code>.",
         "Замените текст внутри вашего элемента <code>p</code> первыми двумя словами текста <code>kitty ipsum</code>: <code>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</code>"
-      ],
+      ],  
+      "titleFR": "Remplissez les blancs avec du faux-texte",
+      "descriptionFR": [
+        "Les développeurs Web utilisent traditionnellement du <code>lorem ipsum</code> comme texte de substitution. Le texte lorem ipsum provient d'un extrait de Cicero, de la Rome antique.",
+        "Le texte lorem ipsum a été utilisé comme faux-texte par des typographes depuis le 16<sup>ème</sup> siècle et cette tradition se poursuit sur le Web.",
+        "Enfin, 5 siècles est suffisament long. Étant donné que nous fabriquons une applications de photos de chats, utilisons plutôt du texte <code>Kitty ipsum</code>.",
```
Comment (Tagada85) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Small typo : 'applications' should read 'application'

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 4]
```diff
@@ -418,6 +473,15 @@
         "<code>&#60;h2 style=\"color: blue\"&#62;CatPhotoApp&#60;/h2&#62;</code>",
         "Измените стиль вашего элемента <code>h2</code> таким образом, чтобы текст элемента стал красным."
       ],
+      "titleFR": "Changer la couleur du texte",
+      "descriptionFR": [
+        "Changeons maintenant la couleur de quelque uns de nos textes.",
```
Comment (Tagada85) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Small typo : 'quelque uns' should read 'quelques-uns'

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 5]
```diff
@@ -486,6 +550,18 @@
         "Обратите внимание, что важно наличие открывающих и закрывающих фигурных скобок (<code>{</code> и <code>}</code>) вокруг стиля каждого элемента. Также вам необходимо убедиться в том, что стиль элемента присвоен внутри элемента <code>style</code>. В завершении, убедитесь, что строка объявления каждого элемента стиля должна заканчиваться точкой с запятой.",
         "Удалите атрибут стиль вашего элемента <code>h2</code> и взамен создайте CSS-элемент <code>style</code>. Добавьте необходимый CSS, чтобы все элементы <code>h2</code> стали синими."
       ],
+      "titleFR": "Utiliser les sélecteurs CSS pour styliser des éléments",
+      "descriptionFR": [
+        "Avec CSS, il y a des centaines de <code>propriétés</code> que vous pouvez utliser pour changer l'apparence d'un élément dans votre page.",
+        "Quand vous avez entré <code>&#60;h2 style=\"color: red\"&#62;CatPhotoApp&#60;/h2&#62;</code>, vous donniez à cet élément <code>h2</code> uniquement, un style <code>inline</code>.",
+        "C'est une des façon d'ajouter un style à un élément, mais une meilleure approche est d'utiliser <code>CSS</code>, accronyme de <code>Cascading Style Sheets</code>.",
```
Comment (Tagada85) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Small typo : 'façon' should read 'façons'

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 6]
```diff
@@ -486,6 +550,18 @@
...
       ],
+      "titleFR": "Utiliser les sélecteurs CSS pour styliser des éléments",
+      "descriptionFR": [
+        "Avec CSS, il y a des centaines de <code>propriétés</code> que vous pouvez utliser pour changer l'apparence d'un élément dans votre page.",
+        "Quand vous avez entré <code>&#60;h2 style=\"color: red\"&#62;CatPhotoApp&#60;/h2&#62;</code>, vous donniez à cet élément <code>h2</code> uniquement, un style <code>inline</code>.",
+        "C'est une des façon d'ajouter un style à un élément, mais une meilleure approche est d'utiliser <code>CSS</code>, accronyme de <code>Cascading Style Sheets</code>.",        
+        "Au sommet de votre code, créez un élément <code>style</code> comme ceci :",
+        "<blockquote>&#60;style&#62;<br>&#60;/style&#62;</blockquote>",                
+        "À l'intérieur de cet élément style, vous pouvez créer des <code>sélecteurs CSS</code> pour tous les élément <code>h2</code>. Par exemple, si vous voulez que tous les éléments <code>h2</code> soient en rouge, votre élément style ressemblerait à ceci :",
```
Comment (Tagada85) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Small typo : 'tous les élément' should read 'tous les éléments'

**Comment by mizujin** [comment]
Oh this is bad -_- I wanted to squash my stuff back to 1 commit but it seems it introduced past commits... is there any safe way to undo this safely ?

**Comment by atjonathan** [comment]
#### LGTM :+1: 

We will merge for now and if something is incorrect, it can be changed later. Thanks for your PR @mizujin :+1:

**Pull request merged**